include/miopen/miopen.h

```diff
@@ -2738,6 +2738,28 @@ miopenBatchNormalizationForwardTraining(miopenHandle_t handle,
                                         void* resultSaveMean,
                                         void* resultSaveInvVariance);
 
+MIOPEN_EXPORT miopenStatus_t
+miopenBatchNormalizationForwardTraining_V2(miopenHandle_t handle,
```

Comment:
since C does not support function overloading had to create `miopenBatchNormalizationForwardTraining_V2` for `miopenBatchNormalizationForwardTraining`